src/components/orders/OrderDetails/index.tsx

```diff
@@ -119,6 +119,18 @@ const tabItems = (
   ]
 }
 
+/**
+ * Get the order with txHash set if it has a single trade
+ *
+ * That is the case for closed orders, fill or kill or partial fill that has a single trade
+ */
+function getOrderWithTxHash(order: Order | null, trades: Trade[]): Order | null {
```

Comment:
ubernit: technically is not a "get" method, you are returning the original order, but with some additional info. Not sure how to call this, but it felt strange seeing the implementation.

Probably, just ignore me in this one since I'm not suggesting a better name.

Reply:
buildOrderWithTxHash?